src/test/java/org/kainos/ea/integration/JobRoleIntegrationTest.java

```diff
@@ -0,0 +1,30 @@
+package org.kainos.ea.integration;
+
+import io.dropwizard.configuration.ResourceConfigurationSourceProvider;
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import io.dropwizard.testing.junit5.DropwizardExtensionsSupport;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.kainos.ea.DropwizardWebServiceApplication;
+import org.kainos.ea.DropwizardWebServiceConfiguration;
+import org.kainos.ea.model.JobRole;
+
+import java.util.List;
+
+@ExtendWith(DropwizardExtensionsSupport.class)
+public class JobRoleIntegrationTest {
+    static final DropwizardAppExtension<DropwizardWebServiceConfiguration> APP = new DropwizardAppExtension<>(
+            DropwizardWebServiceApplication.class, null,
+            new ResourceConfigurationSourceProvider()
+    );
+
+    @Test
+    void test_1getJobRoles_shouldReturnListOfJobRoles() {
+        List<JobRole> response = APP.client().target("http://localhost:8080/api/job_roles")
```

Comment:
don't hardcode 'http://localhost:8080' instead pass it in as an environment variable so you can set it to the AWS URL in the future